setup.py

```diff
@@ -88,7 +78,9 @@
         "tqdm >= 4.14",
     ],
     extras_require={
-        'with-blake2': blake2_requires,
+        'with-blake2': [
```

Comment:
I believe this syntax is less compatible than:

```
    'with-blake2:sys_version<"3.6"...': ['pyblake2']
```

which is less compatible than:

```
    'with-blake2:sys_version=="3.5" or sys_version=="2.7" or ...': ['pyblake2']
```

The last version supports very old versions of setuptools. Personally, I'm fine with the proposed change - we should just recognize that it's increasing the minimum setuptools version required to build from sdist... and should probably declare that in pyproject.toml.